Calibration/IsolatedParticles/test/proto_runIsoTrig_cfg.py

```diff
@@ -3,13 +3,15 @@
 
 process = cms.Process("Demo")
 
-process.load('Configuration/Geometry/GeometryIdeal_cff')
-process.load('Configuration/StandardSequences/MagneticField_38T_cff')
-process.load('TrackingTools/TrackAssociator/DetIdAssociatorESProducer_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_cff')
+process.load('TrackingTools.TrackAssociator.DetIdAssociatorESProducer_cff')
 process.load("FWCore.MessageService.MessageLogger_cfi")
 process.load("Calibration.IsolatedParticles.isoTrig_cfi")
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
-process.GlobalTag.globaltag = 'FT_R_53_V6::All' ## July15ReReco Run2012A & B
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['run2_mc']
```

Comment:
You are loading event data from Run1 PromptReco. Maybe you should use a data key.